tutorecommerce/templates/ecommerce/apps/tutor.py

```diff
@@ -61,6 +78,22 @@
 COMPRESS_OFFLINE = True
 
 PAYMENT_PROCESSOR_CONFIG = {
-    "openedx": json.loads("""{{ ECOMMERCE_PAYMENT_PROCESSORS|tojson(indent=4) }}""")
+    "edx": json.loads("""{{ ECOMMERCE_PAYMENT_PROCESSORS|tojson(indent=4) }}""")
```

Comment:
I think it's better if we keep the "openedx" naming convention. "edx" usually refers to edx.org.